### PR TITLE
Add Power Optimize concept

### PR DESCRIPTION
**Pull request by erikbosch**
This is a power optimization concept based on discussions in the COVESA power optimization project

### CODE REVIEW

**Comment by erikbosch** [comment]
Meeting notes:

- To be discussed at power optimize meeting tomorrow

**Comment by UmangSharmaMobis** [comment]
Please find power optimization group meeting update:

1.  The signal can be added to a branch by including this file, like:
 # "#include include/PowerOptimize.vspec"
so that also full fill the requirement of upcoming signal.
2. it seems fit to merge.

File: spec/include/PowerOptimize.vspec [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+##
+# Power Optimization
+##
+
+  
+PowerOptimizeLevel:
+  datatype: uint8
+  type: actuator
+  min: 1
+  max: 10
```
Comment (erikbosch) [inline spec/include/PowerOptimize.vspec:105]: Meeting notes: Info from EV Power optimize group, shall be refactored, 0 =no power optimize, 10= maximum. If redefined no problem to be merged.

Way forward:

- Erik to update PR
- Mobis/Umang to review after change
- If OK, Erik can merge

Comment (erikbosch) [reply spec/include/PowerOptimize.vspec:105]: PR updated, @UmangSharmaMobis please review

**Review by UmangSharmaMobis — APPROVED** [review]
Given definition is fine. it can be merge.

**Pull request merged**